docs/api/lock-stores.rst

```diff
@@ -0,0 +1,67 @@
+:desc: Messages that are being processed lock Rasa for a given conversation ID to
+  ensure that multiple incoming messages for that conversation do not interfere with
+  each other. Rasa provides multiple implementations to maintain conversation locks.
+
+.. _lock-stores:
+
+Lock Stores
+===========
+
+.. edit-link::
+
+Rasa uses a ticket lock mechanism to ensure that incoming messages for a given
+conversation ID are processed in the right order, and locks conversations while
+messages are actively processed. This means multiple Rasa servers can
+be run in parallel as replicated services, and clients do not necessarily need to
+address the same node when sending messages for a given conversation ID.
+
+.. contents::
+
+InMemoryLockStore (default)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:Description:
+    ``InMemoryLockStore`` is the default lock store. It maintains conversation locks
+    within a single process.
+
+    .. note:: This lock store should not be used when multiple Rasa servers are run
```

Comment:
is the syntax correct here? might need a newline (could also be that the github syntax highlighting is off)

Reply:
👍